docs/getting-started/part-3/custom-webservice.md

```diff
@@ -200,6 +200,8 @@ This logic is implemented in the next code snipped and should replace the code c
 }
 
 ```
+Note, in the *Init* case, the pre-defined **schema** (see the `ext-departures` dataObject definition in the config) is used for the creation of an empty/dummy json file, which then gets converted to the DataFrame. 
```

Comment:
```suggestion
Note, in the *Init* phase, the pre-defined **schema** (see the `ext-departures` dataObject definition in the config) is used to create an empty/dummy json file, which then gets converted to the DataFrame.
```

Reply:
@zzeekk I think this is not strictly true. We don't create an empty Json file, we generate a DataFrame on the fly with the given schema, correct?

Reply:
I am happy if we can describe it in a correct way. I only want to point out that the defined schema comes in place here. This is the important bit for me. 
Thanks for finding to correct description of the present command. 
```Scala
Seq[String]().toDF("responseString")                                                                                                                                                                                                            .select(from_json($"responseString", DataType.fromDDL(schema)).as("response"))                                                                                                                                                            .select(explode($"response").as("record"))                                                                                                                                                                                                .select("record.*")                                                                                                                                                                                                                       .withColumn("created_at", current_timestamp()) 
```